src/common/transformations/src/transformations/mlir/op/matmul.cpp

```diff
@@ -29,8 +29,24 @@ struct ConvertMatMul {
         auto empty = builder.create<tensor::EmptyOp>(loc, outType, dynamic_dimensions);
         auto zero = getConstant(builder, ov_output_element_type, 0);
         auto fill = builder.create<linalg::FillOp>(loc, mlir::ValueRange{zero}, mlir::ValueRange{empty});
-        // TODO: Add other variants of transpose_a/transpose_b
-        auto matmul = builder.create<linalg::MatmulTransposeBOp>(loc, mlir::ValueRange{inputs[0], inputs[1]}, mlir::ValueRange{fill.getResult(0)});
```

Comment:
So if you put this line back it starts working for `transpose_b=true` mode, right?